src/main.rs

```diff
@@ -65,6 +65,13 @@ fn main() -> Result<()> {
                         .takes_value(true)
                         .multiple_values(true),
                 )
+                .arg(
+                    Arg::new("platform")
+                        .long("platform")
+                        .help("Set target platform for your output image")
+                        .takes_value(true)
+                        .multiple_values(true),
```

Comment:
Why can this take multiple values?

Reply:
Because you can build for multiple platforms at once

Reply:
I believe for building multiple platform, we will also have to pass in `--push` and `--load`.